kythe/go/extractors/govname/govname.go

```diff
@@ -207,7 +207,7 @@ func ForStandardLibrary(importPath string) *spb.VName {
 // This includes the "golang.org" corpus but excludes the "golang.org/x/..."
 // extension repositories.  If v == nil, the answer is false.
 func IsStandardLibrary(v *spb.VName) bool {
-	return v != nil && (v.Language == "go" || v.Language == "") && v.Corpus == golangCorpus
+	return v != nil && (v.Language == "go" || v.Language == "") && v.Corpus == GolangCorpus
```

Comment:
It does, I added a comment noting that, but open to suggestions to better handle this. The behavior should be right, but `IsStandardLibrary()` will no longer evaluate to true when `--use_compilation_corpus_as_default` is enabled.

Reply:
This would break all of the callers then.  Flags like `--emit_standard_libs` and `--only_emit_doc_uris_for_standard_libs` will no longer function.

I'm not sure of the best way to handle this other than threading VName provenances everywhere.

Reply:
One possibility *might* be to continue using "golang.org" for these everywhere, but rewrite the corpus in `writeFact()` and `writeEdge()` before actually emitting. Not sure if that would break other things though

Reply:
I'd also need to change `emitCode()` to rewrite  the `Link` field before the rendered marked source is passed to `writeFact()`